.github/workflows/kserve_gpu_tests.yml

```diff
@@ -0,0 +1,45 @@
+name: KServe GPU Nightly Tests
+
+on:
+  workflow_dispatch:
+  # runs everyday  at 5:15am
+  schedule:
+    - cron:  '15 5 * * *'
+
+jobs:
+  kserve-gpu-tests:
+    # runs-on: [self-hosted, regression-test-gpu
```

Comment:
looks like this needs to be uncommented?

Reply:
fixed